view/jpa/src/test/kotlin/io/holunda/polyflow/view/jpa/JpaPolyflowViewServiceDataEntryITest.kt

```diff
@@ -333,6 +333,31 @@ internal class JpaPolyflowViewServiceDataEntryITest {
     assertThat(result.payload.elements.map { it.entryId }).containsExactly(id2) // id4 is not found by correlation to id2, due to property
   }
 
+  @Test
+  fun `should anonymize data entry`() {
+    jpaPolyflowViewService.on(
+      event = DataEntryAnonymizedEvent(
+        entryType = "io.polyflow.test",
+        entryId = id4,
+        anonymizedUsername = "ANONYMOUS",
+        excludedUsernames = listOf("SYSTEM"),
+        anonymizeModification = Modification(
+          time = OffsetDateTime.ofInstant(now, ZoneOffset.UTC),
+          username = "SYSTEM",
```

Comment:
Got you. You are cheating. Please modify this test to create a anonymization event from "KERMIT" user... This will fail the test (and then address the comment in the implementation regarding the order).

Reply:
I still would argue that that would be the correct outcome ;P